src/index.html

```diff
@@ -28,14 +28,16 @@ <h1>Today's To Do</h1>
             <img src="./assets/return.svg" alt="return icon" class="return" />
           </div>
         </form>
-        <hr />
-        <ul class="todo__list">
-          <!-- Dynamicaly Generated -->
-        </ul>
+        <!-- <hr /> -->
+        <div class="list__container">
+          <ul class="todo__list">
+            <!-- Dynamicaly Generated -->
+          </ul>
+        </div>
       </header>
 
       <div class="footer">
-        <h3>Clear all completed</h3>
+        <h3 class="clearMarked">Clear all completed</h3>
```

Comment:
- [ ] Great job so far your project look great :clap: whatever the `Clear all completed` should be a button with a click EventListener which remove All hardcoded items from the tasks array, kindly create a function that remove all hardcoded items from the tasks array when you click to the `Clear all completed`